src/features/surveys/components/surveyForm/OptionsQuestion.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from 'react';
+import {
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  FormLabel,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  Select,
+} from '@mui/material';
+import {
+  ZetkinSurveyOption,
+  ZetkinSurveyOptionsQuestionElement,
+} from 'utils/types/zetkin';
+
+const OptionsQuestion: FC<{ element: ZetkinSurveyOptionsQuestionElement }> = ({
+  element,
+}) => {
+  return (
+    <FormControl>
+      <FormLabel
+        id="demo-radio-buttons-group-label"
+        style={{
+          color: 'black',
+          fontSize: '1.5em',
+          fontWeight: '500',
+          marginBottom: '0.5em',
+          marginTop: '0.5em',
+        }}
+      >
+        {element.question.question}
+      </FormLabel>
+      {element.question.response_config.widget_type === 'checkbox' && (
```

Comment:
I think it's technically possible for `widget_type` to be `undefined` in which case we need to default to one of these three UI options, right? I think `select` is the default in Gen2.

Reply:
Fixed in 95b059ca496dee0730e1ab2b7e10005327698622!